src/assets/_project/_blocks/utils/qg-date-ie-support.js

```diff
@@ -0,0 +1,10 @@
+function browserSupportsDateInput () {
+  var i = document.createElement('input');
+  i.setAttribute('type', 'date');
+  return i.type !== 'text';
```

Comment:
when do you cleanup said element i

Reply:
Garbage collection is automatic in JS https://javascript.info/garbage-collection

And I am checking it once when a browser load,  I am checking date is present or not. 
I tested on browserstack, it works fine